### PR TITLE
Adding sample of Halles des Vosges, Belfort, France

### PR DESCRIPTION
**Pull request by amarrocq**
- .mid and .mif files
- .ext files
- image of sector subdivision 
- image of front and back of the model

### CODE REVIEW

**Comment by amarrocq** [comment]
I opened an issue #4 with suggestions to work on developper experience using this format.

File: sample/halles_des_vosges/belfort.xml [hunk 1]
```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (yobeonline) [inline sample/halles_des_vosges/belfort.xml]: This file is not related to the ext file format. Please remove it.

**Review by yobeonline — CHANGES_REQUESTED** [review]
The renaming of the la Defense sample breaks many hyperlinks in the README.md file at the root of the repository. Please update the file.

**Comment by amarrocq** [comment]
*.xml* removed and hyperlinks should be fixed

File: sample/halles_des_vosges/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Halles des Vosges, Belfort, France
+
+This sample is a demonstration of multi-material building and slopped roof.
+
+The construction method is based on the building divided in sectors. These sectors are specified by their angles, and have no fixed amount of sides, in the *.mif* and *.mid* files. The slopes are specified in each *.ext* file. In this case 18 files are sufficient to describe all 28 sectors.
+
+![](./sectors.png)
+\
+*Figure 1 : Sectors*
```
Comment (yobeonline) [inline sample/halles_des_vosges/README.md]: Please use the same typographical convention as in the README.md file at root. Ie. *Figure 1 — Sectors.*

File: sample/halles_des_vosges/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+With the *.ext* file format you can reduce to a certain minimum the number of files you need, because this building specific building has many slopped roof, all different slope needs to have its own description.
+\
+\
```
Comment (yobeonline) [inline sample/halles_des_vosges/README.md]: Leave the management of interparagraph spaces to the stylesheet. Remove the `\` markup tags.

File: sample/halles_des_vosges/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Halles des Vosges, Belfort, France
+
+This sample is a demonstration of multi-material building and slopped roof.
+
+The construction method is based on the building divided in sectors. These sectors are specified by their angles, and have no fixed amount of sides, in the *.mif* and *.mid* files. The slopes are specified in each *.ext* file. In this case 18 files are sufficient to describe all 28 sectors.
+
+![](./sectors.png)
```
Comment (yobeonline) [inline sample/halles_des_vosges/README.md]: How about using the svg directly here and removing the png version from the repository? Besides, in the preview, it is way too big.

File: sample/halles_des_vosges/README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+\
+![](./halles_front.png)
+\
+*Figure 2 : Front of the building*
+\
+\
+\
+![](./halles_back.png)
+\
+*Figure 3 : Back of the building*
```
Comment (yobeonline) [inline sample/halles_des_vosges/README.md]: These two pictures should be gathered in a single picture, referring them as figure 2 let and right.

File: sample/halles_des_vosges/README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# Halles des Vosges, Belfort, France
+
+This sample is a demonstration of multi-material building and slopped roof.
+
+The construction method is based on the building divided in sectors. These sectors are specified by their angles, and have no fixed amount of sides, in the *.mif* and *.mid* files. The slopes are specified in each *.ext* file. In this case 18 files are sufficient to describe all 28 sectors.
```
Comment (yobeonline) [inline sample/halles_des_vosges/README.md]: I don't understand what "their angle" refers to. Same for "no fixed amount of side". Please clarify.

File: sample/halles_des_vosges/README.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+# Halles des Vosges, Belfort, France
+
+This sample is a demonstration of multi-material building and slopped roof.
```
Comment (yobeonline) [inline sample/halles_des_vosges/README.md]: "… is a demonstration of how to model a multi-material building with slopped roof"

File: sample/halles_des_vosges/README.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+The construction method is based on the building divided in sectors. These sectors are specified by their angles, and have no fixed amount of sides, in the *.mif* and *.mid* files. The slopes are specified in each *.ext* file. In this case 18 files are sufficient to describe all 28 sectors.
+
+![](./sectors.png)
+\
+*Figure 1 : Sectors*
+
+With the *.ext* file format you can reduce to a certain minimum the number of files you need, because this building specific building has many slopped roof, all different slope needs to have its own description.
```
Comment (yobeonline) [inline sample/halles_des_vosges/README.md]: What does "building specific building" mean ?

Comment (yobeonline) [reply sample/halles_des_vosges/README.md]: This sentence does not seem to bring any new info compared to the one on line 5.

File: sample/halles_des_vosges/README.md (hunk 6 above)
Comment (yobeonline) [reply sample/halles_des_vosges/README.md]: I believe this paragraph should be rewritten along these lines:

1. describe what the building is (mainly glass walls with concrete bottom). Since we cannot quite use a real life picture of it, maybe it is a good idea to introduce the resulting model (figure 2/3) here to help.
1. explain that the building was split into 28 parts so that each require a single ext file.
1. explain that disjoint parts can actually share the same ext file (give an example).
2. explain that on the contrary maybe some parts had to be split apart because of inner material differences.
3. explain that dedicated parts were provided for the walls because it would otherwise have created a concrete floor/ceiling at midheight.

Do not forget to introduce figures in the main text. A figure is supposed to support text, hence the text should mention them.

**Review by yobeonline — CHANGES_REQUESTED** [review]

**Comment by amarrocq** [comment]
New updated README.md pushed rewritten along given directions.
